src/redux/features/plantings/plantingsSelectors.ts

```diff
@@ -81,19 +81,23 @@ export const selectPlantingProgress = createSelector(
 // selector to search plantings
 export const searchPlantingProgress = createSelector(
   [
-    (state: RootState, query: string, plantingCompleted?: boolean) => selectPlantingProgress(state),
-    (state: RootState, query: string, plantingCompleted?: boolean) => query,
-    (state: RootState, query: string, plantingCompleted?: boolean) => plantingCompleted,
+    (state: RootState, query: string, plantingCompleted?: boolean, plantingSiteId?: number) =>
+      selectPlantingProgress(state),
+    (state: RootState, query: string, plantingCompleted?: boolean, plantingSiteId?: number) => query,
+    (state: RootState, query: string, plantingCompleted?: boolean, plantingSiteId?: number) => plantingCompleted,
```

Comment:
curious what happens if we drop the plantingSiteId? param for these since it is unused